template/src/App.tsx

```diff
@@ -108,6 +110,75 @@ const App = () => {
 
 // *****************************************************************************************************
 // The rest of the file is to set up a react-navigation and react-native-vector-icons demonstration:
+const Stack = createStackNavigator();
+const StackNavigator = () => {
+  const linkTo = useLinkTo();
+  const navigation = useNavigation();
+
+  // Dark mode theming items
+  const isDarkMode = useColorScheme() === 'dark';
+  const primaryColor = isDarkMode ? Colors.darker : Colors.lighter;
+  const accentColor = isDarkMode ? Colors.lighter : Colors.darker;
+  const backgroundStyle = {backgroundColor: primaryColor, flex: 1};
+
+  const NotFound = () => {
+    return (
+      <View style={[backgroundStyle, styles.detailsContainer]}>
+        <Text
+          style={[
+            styles.sectionTitle,
+            {
+              color: accentColor,
+            },
+          ]}>
+          Oh no! This page does not exist.
+        </Text>
+        <View style={styles.spacer} />
+        <Text
+          style={[
+            styles.sectionTitle,
+            {
+              color: accentColor,
+            },
+          ]}>
+          On Android and web, the back button will take you back. On iOS you may
+          swipe to go back.
+        </Text>
+        {navigation.canGoBack() && (
+          <>
+            <View style={styles.spacer} />
+            <Button
+              title="navigation.goBack()"
+              onPress={() => {
+                navigation.goBack();
+              }}
+            />
+          </>
+        )}
+        <View style={styles.spacer} />
+        <Button title="Go to '/'" onPress={() => linkTo('/')} />
+      </View>
+    );
+  };
+
+  return (
+    <Stack.Navigator
+      initialRouteName="App"
+      screenOptions={{
+        // TODO - investigate why setting width/height is needed on web but not mobile.
+        // Without it, the card gets height of content in largest tab and overflows with no scroll?
+        cardStyle: {
+          width: Dimensions.get('window').width,
+          height: Dimensions.get('window').height,
+        },
```

Comment:
This really bothers me, but I was unable to find a way around it. I suppose it could get a `Platform.OS === 'web'` wrapper as mobile doesn't need it but it also works fine on all at once.